h/static/scripts/tests/util/submit-form-test.js

```diff
@@ -0,0 +1,98 @@
+'use strict';
+
+var submitForm = require('../../util/submit-form');
+
+/**
+ * Fake `fetch()` implementation.
+ *
+ * This does not mock the global `window.fetch` function but provides a mock
+ * via `this.fetch` which can be passed as a dependency to functions that make
+ * network requests.
+ */
+class FakeFetch {
```

Comment:
Done. I did look at `fetch-mock` in the past and dismissed it but I can't recall why. I took another look again just now and it worked fine.